tests/cross_functional/system_test/test_full_cluster_health.py

```
@@ -115,102 +115,203 @@ def delete_pods(self):
+        logger.info("All Ceph pods verified running after MGR node restart")
 
     def restart_ocs_operator_node(self):
         """
         Restart node that runs OCS operator pod
         """
-
+        logger.info("Identifying OCS operator pod and its node")
         pod_obj = pod.get_ocs_operator_pod()
         node_obj = pod.get_pod_node(pod_obj)
+        logger.info(
+            f"OCS operator pod '{pod_obj.name}' running on node '{node_obj.name}'"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)